src/main/java/mate/academy/bookshop/validation/Isbn.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.bookshop.validation;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Constraint(validatedBy = IsbnValidator.class)
+@Target({ElementType.PARAMETER, ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Isbn {
```

Comment:
ISBN validation annotation already exists in bean validation package, so it's redundant to create your own. But for practicing of implementation of your custom validators you can keep it.